package.json

```diff
@@ -9,18 +9,19 @@
   ],
   "scripts": {
     "upload": "node-riffraff-artifact",
-    "test:build": "webpack --config webpack.config.js --config-name server --env.test",
+    "test:build": "npm run build:client:prod; webpack --config webpack.config.js --config-name server --env.test",
     "test": "npm run test:build && jest --verbose",
     "test:coverage": "npm run test && open coverage/index.html",
     "lint": "node_modules/eslint/bin/eslint.js -c config/.eslintrc.js ./ --ext '.ts,.tsx'",
     "watch:server": "webpack --config webpack.config.js --config-name server --env.watch",
     "watch:client": "webpack-dev-server --config webpack.config.js --config-name client",
-    "watch": "npm run watch:server & npm run watch:client",
+    "watch": "npm run build:client; npm run watch:server; npm run watch:client",
```

Comment:
This initial `npm run build:client;` is for the server to read `dist/assets/manifest.json`